roles/network_plugin/cilium/templates/cilium-operator/deploy.yml.j2

```diff
@@ -145,8 +145,13 @@ spec:
             labelSelector:
               matchLabels:
                 io.cilium/app: operator
+{% if cilium_operator_tolerations is defined %}
+      tolerations:
+        {{ cilium_operator_tolerations | list | to_nice_yaml(indent=2) | indent(8) }}
+{% else %}
       tolerations:
         - operator: Exists
+{% endif %}
```

Comment:
You should instead ditch the conditional and define in the roles default:
```
cilium_operator_tolerations:
- operator: Exists
```

Reply:
I've implemented the suggestion, thanks!